MessageBird/Objects/Conversations/Webhook.cs

```diff
@@ -0,0 +1,42 @@
+using System;
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+using Newtonsoft.Json.Serialization;
+
+namespace MessageBird.Objects.Conversations
+{
+    public enum WebhookEvent
+    {
+        [EnumMember(Value = "conversation.created")]
+        ConversationCreated,
+        [EnumMember(Value = "conversation.updated")]
+        ConversationUpdated,
+        [EnumMember(Value = "message.created")]
+        MessageCreated,
+        [EnumMember(Value = "message.updated")]
+        MessageUpdated,
+    }
+    
+    public class Webhook : IIdentifiable<string>
+    {
+        [JsonProperty("id")]
+        public string Id {get;set;}
+        
+        [JsonProperty("channelId")]
+        public string ChannelId {get;set;}
+        
+        [JsonProperty("url")]
+        public string Url {get;set;}
+        
+        [JsonProperty("events", ItemConverterType = typeof(StringEnumConverter))]
+        public List<WebhookEvent> Events {get;set;}
```

Comment:
Any reason this is a generic list rather than a custom type (like with `MessageList` or `WebhookList`)?

Reply:
This is not a separate resource